src/components/routerView.vue

```diff
@@ -48,7 +51,7 @@
     }
 
     const component = getComponent(match)
-    const props = getProps(match)
+    const props = propStore.getProps(route, name)
```

Comment:
Coming back here after looking at `PropStore. Since this uses the `route` and not the `match` depth isn't factored in here right? Its going to get the props for the first component by the matching name right? 

We might not have good tests to accurately reflect this logic.